tests/sca/test_virtwho_sca.py

```diff
@@ -0,0 +1,71 @@
+# coding:utf-8
+from virt_who import *
+from virt_who.base import Base
+from virt_who.register import Register
+from virt_who.testing import Testing
+
+
+class VirtWhoSCATestCase(Testing):
+
+    @classmethod
+    def setUpClass(cls):
+        super(VirtWhoSCATestCase, cls).setUpClass()
+        register_config = cls.get_register_config(cls())
+        if 'satellite' in register_config['type']:
+            cls.satellite_sca_set(cls(), cls.ssh_host(cls()), register_config, enable=True)
+        else:
+            cls.stage_sca_set(cls(), cls.ssh_host(cls()), register_config, enable=True)
```

Comment:
Maybe it's better to define a function ```sca_set()``` for the both satellite and stage using. Will perfect it in further prs